kkanbu/C_Obstacle[] camera_obstacle
